frontend/src/scenes/funnels/funnelLogic.test.ts

```diff
@@ -500,7 +500,7 @@ describe('funnelLogic', () => {
 
         it('Deselecting all returns empty result', async () => {
             await expectLogic(logic, () => logic.actions.setPropertyNames([]))
-                .toFinishListeners()
+                .toDispatchActions(logic, ['loadPropertyCorrelationsSuccess'])
```

Comment:
Is this testing this change? 

Shouldn't there be a check / expect for the values of `propertyNames`?

Reply:
Ah, no, not testing this change! (making things more explicit here, because logic isn't re-inited between tests). I think it was subsumed into the test above, which runs `setPropertyNames`